website/docs/kubernetes/deploying-a-container-to-kubernetes.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 10
+title: Deploying a container
+description: Deploying a container to Kubernetes 
+keywords: [podman desktop, podman, containers, migrating, kubernetes]
+tags: [migrating-to-kubernetes]
+---
+
+# Deploying a container to Kubernetes
+
+You can deploy a container to Kubernetes with Podman Desktop.
```

Comment:
"to a kubernetes cluster"? 
Maybe we should have a pre-requisites section?
and there we can say "if you don't have a kubernetes cluster, you can get one from the Kind extension" and include a link